hack/build_lua_deps.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export BUILD_PATH=rootfs/etc/nginx/lua/vendor
```

Comment:
Can you call this `DESTDIR`? To be consistent with https://github.com/openresty/lua-resty-lock/blob/master/Makefile#L13.